README.md

```diff
@@ -95,4 +95,18 @@ cargo run --features=try-runtime try-runtime on-runtime-upgrade live --uri wss:/
 or against the Basilisk testnet on Rococo
 ```
 cargo run --release --features=try-runtime try-runtime on-runtime-upgrade live --uri wss://rococo-basilisk-rpc.hydration.dev:443
+```
+
+### Chopsticks simulations
+`Chopsticks` can be used to dry-run any transaction in parallel reality of any Substrate network.
+
+Setting up the repo:
+```bash
+git clone --recurse-submodules https://github.com/AcalaNetwork/chopsticks.git && cd chopsticks
+yarn
+yarn build-wasm
+```
+To run Kusama-Basilisk-Karura setup use configs from `launch-configs/chopsticks` and run
+```bash
+npx @acala-network/chopsticks@0.3.11 xcm --relaychain=configs/kusama.yml --parachain=configs/basilisk.yml --parachain=configs/karura.yml
```

Comment:
wouldn't this be `--parachain=../Basilisk-node/launch-configs/chopsticks/basilisk.yml` in case chopsticks and Basilisk are checked out side-by-side?

Reply:
yep it would. seems @mrq1911 works on a customized fork of chopsticks for GC so i will wait with amend